fingerprint_verification.py

```diff
@@ -1,44 +1,148 @@
 ######################################################################
 # Names of any others you worked with:
+# Received some help/advice from Hailee & Bryant (Namely my trouble with calling the correct file, then with some of the math for shifted_check)
+# Additionally, some use of reddit/google & my notes for certain built-ins (i.e. splitlines(), len, etc.)
 # AI transcript if used:
 # Any extensions done:
 ######################################################################
 
+# open and read the file then return the content as a string
 def read_print(filename):
     """Reads the given print and returns a dictionary with all info."""
-    # Your turn!
-
-
-
-
-
-
-# You can add your various check functions below
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    with open(filename, 'r') as fh:
+        # splitlines after read to remove \n
+        lines = fh.read().splitlines()
+    # the string of characters being the fingerprint
+    # currently the "name, width, height" comes according to file
```

Comment:
I'd recommend trying to match indentation of comments with the code they are describing. It just tends to eliminate eye-strain of having to parse lots of changes in indentation.